examples/resources/bloxone_dhcp_fixed_address/resource.tf

```diff
@@ -0,0 +1,55 @@
+resource "bloxone_ipam_ip_space" "example" {
+  name    = "example"
+  comment = "Example IP space created by the terraform provider"
+  tags = {
+    location = "site1"
+  }
+}
+
+resource "bloxone_ipam_address_block" "example" {
+  address = "192.160.0.0"
+  cidr    = 16
+  space   = bloxone_ipam_ip_space.example.id
+  comment = "Example Address Block created by the terraform provider"
+  tags = {
+    location = "site1"
+  }
+}
+
+resource "bloxone_ipam_subnet" "example" {
+  name    = "example"
+  space   = bloxone_ipam_ip_space.example.id
+  address = "192.168.1.0"
+  cidr    = 24
+  comment = "Example Subnet created by the terraform provider"
+  tags = {
+    location = "site1"
+  }
+}
+
+resource "bloxone_ipam_range" "example" {
+  start = "192.168.1.15"
+  end   = "192.168.1.30"
+  space = bloxone_ipam_ip_space.example.id
+
+  # Other optional fields
+  name    = "example"
+  comment = "Example Range created by the terraform provider"
+  tags = {
+    location = "site1"
+  }
+  depends_on = [bloxone_ipam_subnet.example]
+}
+
```

Comment:
do you need range for this example?

Reply:
the bloxone_ipam_range is not removed yet

Reply:
updated